### ISSUE TITLE
Vulkan:Multiple problems after #4908

### ISSUE THREAD

**Issue by vsub** [RPCS3/rpcs3#5508]
I am using RPCS3 0.0.5-7657-6c10cb2e and this happens only while using Vulkan

Asuras Wraith
Ar noSurge
Atelier Escha & Logy
Atelier Shalie
The music keeps playing but the game pauses(Jump=>minimize=>you will stay there until you restore the window...controller inputs are ignored)

Persona 5
The emulation pauses and you can stat it again but only the audio start playing and nothing else is reacting
F {RSX [0x000711c]} RSX: class std::runtime_error thrown: Verification failed (e=0x7f): 
(in file c:\projects\rpcs3\rpcs3\emu\rsx\vk\vkoverlays.h:229)

I can see how the vram usage goes up fast higher and higher until it reaches 3GB and then,the games below

Project Diva F
F {RSX [0x01e6e28]} RSX: class std::runtime_error thrown: Assertion Failed! Vulkan API call failed with unrecoverable error: Out of video memory (VRAM) (VK_ERROR_OUT_OF_DEVICE_MEMORY)
(in file c:\projects\rpcs3\rpcs3\emu\rsx\vk\vkhelpers.h:253)

Tales of Graces F
F {RSX [0x093ab68]} RSX: class std::runtime_error thrown: Assertion Failed! Vulkan API call failed with unrecoverable error: Out of video memory (VRAM) (VK_ERROR_OUT_OF_DEVICE_MEMORY)
(in file c:\projects\rpcs3\rpcs3\emu\rsx\vk\vkhelpers.h:253)

Tales of Vesperia
F {RSX [0x0085b24]} RSX: class std::runtime_error thrown: Assertion Failed! Vulkan API call failed with unrecoverable error: Out of video memory (VRAM) (VK_ERROR_OUT_OF_DEVICE_MEMORY)
(in file c:\projects\rpcs3\rpcs3\emu\rsx\vk\vkhelpers.h:253)

Unaffected games
BLEACH: Soul Resurreccion
Dragon's Crown
Enchanted Arms
ETERNAL SONATA
Hard Corps: Uprising
Ar Tonelico Qoga

Win10,i7-6700HQ,GTX1060 3GB and it doesn't matter what settings you are using other than using vulkan

**Comment by vsub** [comment]
Nope,I just found the one responsible https://github.com/RPCS3/rpcs3/pull/4908
I only test Project Diva F but
0.0.5-7594 - works fine
0.0.5-7596 - eats all vram and then it crashes

**Comment by psennermann** [comment]
I have experienced the same problem at least with the game Catherine

Win10, i5 7400, gtx1060 3gb

**Comment by kd-11** [comment]
Try with https://github.com/RPCS3/rpcs3/pull/5509 but I do not expect this to do anything. While minimized, nvidia cards do not run through the whole present loop which can possibly leak. This is because nvidia prohibits rendering whilst the main window is minimized (reports max dimensions of 0x0 which is impossible)

**Comment by vsub** [comment]
Sadly no change,all games are acting the same way and yes,
Also why some games work fine and other don't.

**Comment by kd-11** [comment]
While rendering is effectively paused, its does not stop completely. Operations like blit engine and framebuffer readbacks will still invoke compute operations which will consume resources and since there is no frame scope to work with, this can balloon. Its a tough situation here - we cannot stop the operations from happening as graphics will be corrupt when you restore the window and we cannot easily free/reset resources. I'll have to think of a workaround for this at a later time.

**Comment by vsub** [comment]
Well it seems that the 0x0 size it's not the problem
When I use the latest beta and minimize the window,rpcs3 freezes and I get this in Process Hacker for the game window

![2019-01-06_115604](https://user-images.githubusercontent.com/5032540/50735605-d6a82600-11a9-11e9-8821-2ddba9132384.jpg)

And when I try 0.0.5-7594,I also get the exact same results but rpcs3 don't freeze

And also I found another problem so I am changing the title

After #4908 if you pause the emulation and try to move,maximize,minimize the game window,rpcs3 will freeze

**Comment by vsub** [comment]
Sadly no change,all of the mentioned games are acting the same way

**Comment by vsub** [comment]
Any update on this?
It still happens on 0.0.5-7758

**Comment by kd-11** [comment]
Retest

**Comment by vsub** [comment]
I didn't do much testing(just starting the game and then minimize it)but all games except Persona 5 are acting the same way.
Only Persona 5 seems to be fine now

**Comment by psennermann** [comment]
For me the problem is still there (0.0.6.7786)

**Comment by uaqlover** [comment]
problem is still there (0.0.6.7786)

**Comment by vsub** [comment]
I am not sure how helpful this will be but,if I disable the game window minimize button with Autohotkey
```
Loop
{
WinWaitActive,FPS ahk_exe rpcs3.exe
WinSet,Style,-0x20000,FPS ahk_exe rpcs3.exe
WinWaitClose,FPS ahk_exe rpcs3.exe
}
```
and try to minimize,then the problem is not triggered.
You don't really have many "normal" other ways do minimize the window except  Win+D or "Show Desktop" button on the bottom right.
Those two don't apply WM_MINIMIZE to the window so the problem is not triggered but if you actually send the minimize command to the window(from the task manager for example)even if the button is disabled,the WM_MINIMIZE style will be applied and rpcs3 will freeze

**Comment by vsub** [comment]
Still no change on the latest beta(0.0.6-8034)with the latest video drivers(430.39)
Minimizing the game window eats all of the vram and then rpcs3 freezes

**Comment by vsub** [comment]
Fixed since #6069
